app/src/main/java/com/framgia/toeic/screen/grammar_test/GrammarTestActivity.java

```diff
@@ -7,26 +7,39 @@
 import android.support.v4.app.Fragment;
 import android.support.v4.view.ViewPager;
 import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.RatingBar;
 import android.widget.TextView;
+import android.widget.Toast;
 
 import com.framgia.toeic.R;
 import com.framgia.toeic.data.model.Grammar;
-import com.framgia.toeic.data.model.GrammarLesson;
+import com.framgia.toeic.data.repository.MarkRepository;
+import com.framgia.toeic.data.source.local.DBHelper;
+import com.framgia.toeic.data.source.local.MarkDatabaseHelper;
+import com.framgia.toeic.data.source.local.MarkLocalDataSource;
 import com.framgia.toeic.screen.base.BaseActivity;
 import com.framgia.toeic.screen.base.DisplayAnswerListener;
 import com.framgia.toeic.screen.base.ShowAnswerListener;
+import com.framgia.toeic.screen.grammar_test.fragment_grammar.GrammarTestFragment;
 
 import java.util.ArrayList;
 import java.util.List;
 
-public class GrammarTestActivity extends BaseActivity implements ShowAnswerListener {
+public class GrammarTestActivity extends BaseActivity
+        implements ShowAnswerListener, View.OnClickListener,
+        GrammarTestContract.View, GrammarTestFragment.OnAnswerChangeListener,
+        ViewPager.OnPageChangeListener {
     static final String EXTRA_LIST_GRAMMAR = "EXTRA_LIST_GRAMMAR";
+    private static final int ID_GRAMMAR = 2;
```

Comment:
cho a hỏi biến này là gì và tại sao giá trị của nó = 2
```suggestion
    private static final int GRAMMAR_ID = 2;
```